core-java-lang-oop-2/.gitignore

```diff
@@ -0,0 +1,26 @@
+*.class
+
+0.*
+
+#folders#
+/target
+/neoDb*
+/data
+/src/main/webapp/WEB-INF/classes
+*/META-INF/*
+.resourceCache
+
+# Packaged files #
+*.jar
+*.war
+*.ear
+
+# Files generated by integration tests
+*.txt
+backup-pom.xml
+/bin/
+/temp
+
+#IntelliJ specific
+.idea/
+*.iml
```

Comment:
This has a lot of stuff in that seems outside of the scope of the example - where did it originate?

May be simpler to just have

```
target/
.idea/
bin/
*.iml
```

Perhaps the equivalent eclipse project files too...?